### ISSUE TITLE
Suggestion - PD Panic Button Physical Item/Use

### ISSUE THREAD

**Issue by ImArteria** [InZidiuZ/op-framework-issue-tracker#3044]
### Assurance

I have read the `README.md`

### Summary

Panic Buttons are really "roleplay" based and used via /tracker and /mes. A lot of people have keyboard binds for /me breaks tracker, /me finds tracker, etc. You also have some people who think they need to attempt to break it. Some cadets have not turned off their tracker before when they should have. I feel a server mechanic should be in place for this. 

Add a tracker, a 0 pound item (would just go on your vest I assume) that is a physical item on your person. If a criminal has a hammer they can break the tracker being near a cop, doing a 5 second animation, and it turns off the tracker. I feel the hammer is a not really used item anymore and would make sense. After you break it, the tracker turns into a "broken" tracker (0 durability). 

Alternatively if you wanted to potentially do it an easier way, just a simple /break_tracker command for crims would suffice with an animation.

tldr; make breaking trackers a mechanic and not a /me thing. Keep /tracker though for cops to manually do it pls. 

### Reason

Would get rid of a gray area within crims/cops that has always existed
Would be good to give crims a mechanic to do this instead of relying on 2 parties.

**Comment by coalaura** [comment]
I feel like this is overly complicated for something that is literally 1 sentence during cadet training
